app/package.json

```diff
@@ -3,14 +3,15 @@
   "version": "1.0.0",
   "description": "The online portal for Industry to apply and track their CIIP applications",
   "scripts": {
-    "pretest": "npm run relay",
+    "pretest": "npm run build",
     "test": "jest",
-    "predev": "npm run relay",
+    "predev": "npm run build",
     "dev": "node server",
-    "prebuild": "npm run relay",
-    "build": "next build",
+    "build": "npm run build:schema && npm run build:relay && npm run build:next",
+    "build:schema": "postgraphile -X -c postgres://localhost/ggircs_dev -s ggircs_portal --export-schema-graphql server/schema.graphql --export-schema-json server/schema.json --classic-ids --sort-export",
```

Comment:
@matthieu-foucault `--sort-export` added here

Reply:
...though I probably shouldn't be hardcoding `postgres://localhost/ggircs_dev`...

Reply:
Running `build:schema` in the `build` script won't work on CI, as postgres is not running in the builder image

Reply:
I see that 02429bb79c4fee264cf3319fe00c11324b27bfd6 fixes this. Can you cherry pick it to this PR?

Reply:
cherry-pick'd it over